matrix-api.c

```diff
@@ -652,7 +652,7 @@ MatrixApiRequestData *matrix_api_sync(MatrixConnectionData *conn,
 
     url = g_string_new(conn->homeserver);
     g_string_append_printf(url,
-            "_matrix/client/r0/sync?access_token=%s&timeout=%i",
+            "_matrix/client/unstable/sync?access_token=%s&timeout=%i",
```

Comment:
what is the reason to switch to `unstable`? It should not be used (anymore)

Reply:
Thanks, krombel - it stabilised since I wrote this (the spec is still a bit odd - it says it's being worked on but has now started using the r0/ path).  I'll update it.